tests/unit/bokeh/core/property/test_datetime.py

```diff
@@ -137,6 +137,9 @@ def test_str(self) -> None:
         prop = bcpd.Datetime()
         assert str(prop) == "Datetime"
 
+class Test_FloatOrDatetime(Test_Datetime):
+    pass
+
```

Comment:
Not sure if this is needed. It seems redundant, but a test was required.

Reply:
A test to add would be to assert that `FloatOrDatetime` is a subclass of `Datetime`